Assets/Scripts/Events/SubscribersOrderer.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Growing.Events
+{
+    public sealed class SubscribersOrderer
+    {
+        public int Order(Type subscriberType)
+        {
+            // todo
```

Comment:
I suppose, that this todo is placed here to create some kind of a dictionary with type as a key and order as value later, but I think we should provide more data when adding todo's, to make it hard to forget, and easy to remember

Reply:
Added logic